docs/server-side/v2/active-rooms/get-peers.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
```

Comment:
shall we update this to List peers to maintain consistency with other list APIs.